docker-wrappers/RWR/README.md

```diff
@@ -0,0 +1,29 @@
+# RWwR Docker image
```

Comment:
Let's try to consistently use either the RWR acronym, for example in the directory names, or the longer random-walk-with-restart. Lowercase "rwr" is okay in the config file for consistency.